src/App.tsx

```diff
@@ -54,6 +54,22 @@ const theme = createTheme({
       },
     },
   },
+  components: {
+    MuiInputLabel: {
+      styleOverrides: {
+        shrink: {
+          "@media (max-width: 600px)": {
+            color: "#F2F0EB !important",
```

Comment:
In order to find a way to style label when input is focused I tried to use MUI theme styling, because I couldn't find a proper way to style it in TextField component using sx property. But when styling label component in theme I found out that my theme styles are overrode by another MUI style which I could not find to restyle. That's why I decided to use "!important" to prioritize my styles, although it's not considered as the best practice. However in my next branch I rewrote component styles in theme is such a way:
` components: {
    MuiInputLabel: {
      styleOverrides: {
        root: {
          "&.Mui-focused": {
            "@media (max-width: 600px)": {
              color: "#F2F0EB",
            },
            "@media (min-width: 600px)": {
              color: "#712B75",
            },
            position: "absolute",
            top: -7,
          },
        },
      },
    },
  },`